ddR/R/ddR.R

```diff
@@ -80,32 +80,40 @@ useBackend <- function(driver, ...) {
 #' @export
 setClass("ddRDriver", representation(DListClass = "character", DFrameClass = "character", DArrayClass = "character", backendName = "character"))
 
-#' Called when the backend driver is initialized.
+#’ @rdname package-deprecated
+#' Initialize backend driver
 #'
-#' @param x The driver object to initialize the backend for.
+#' Used internally by ddR to set up a new backend driver.
+#'
+#' @param driver The driver object to initialize the backend for.
 #' @param ... Other parameters to pass to the initialization routine.
+#' @seealso \code{\link{useBackend}} for the user facing way to set or
+#'      change backends
 #' @export
-setGeneric("init", function(x,...) standardGeneric("init"))
+setGeneric("init", function(driver,...) standardGeneric("init"))
```

Comment:
@lawremi exactly when does this convention apply? Looking at `do_mapply` and `combine` the first arg is `driver`, which seems like a nice descriptive name. So I was changing `init` and others to match. For internal consistency they should either all be `x` or `driver`.